### PR TITLE
auto ocp21333

### PR DESCRIPTION
**Pull request by QiaolingTang**
@anpingli  PTAL, thanks.

Polarion cases: https://url.corp.redhat.com/bfc0843

Log: https://url.corp.redhat.com/3b95aa4

### CODE REVIEW

File: features/logging/4.x/cluster_logging_operator.feature [hunk 1]
```diff
@@ -0,0 +1,33 @@
+@clusterlogging
+Feature: cluster-logging-operator related test
+
+  # @author qitang@redhat.com
+  # @case_id OCP-21333
+  # @case_id OCP-19875
+  # @case_id OCP-24111
+  # @case_id OCP-24578
```
Comment (akostadinov) [inline features/logging/4.x/cluster_logging_operator.feature]: Please check [DOC-1125811](https://mojo.redhat.com/docs/DOC-1125811) for info about the `case_id` tag.

-- saved reply

Comment (akostadinov) [reply features/logging/4.x/cluster_logging_operator.feature]: otherwise looks good as long as your team is happy with it.

Comment (QiaolingTang) [reply features/logging/4.x/cluster_logging_operator.feature]: @akostadinov I missed some checkpoints here, I'll divide this scenario into 2 scenarios. Thanks

**Comment by QiaolingTang** [comment]
@anpingli PTAL if you have time, thanks.
Run log: https://url.corp.redhat.com/31da6ed

**Pull request merged**